src/elements/PolygonAperture.H

```diff
@@ -0,0 +1,321 @@
+/* Copyright 2022-2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Eric G. Stern, Chad Mitchell, Axel Huebl
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_POLYGONAPERTURE_H
+#define IMPACTX_POLYGONAPERTURE_H
+
+
+#include "particles/ImpactXParticleContainer.H"
+#include "mixin/alignment.H"
+#include "mixin/beamoptic.H"
+#include "mixin/thin.H"
+#include "mixin/lineartransport.H"
+#include "mixin/named.H"
+#include "mixin/nofinalize.H"
+
+#include <AMReX_Extension.H>
+#include <AMReX_Math.H>
+#include <AMReX_REAL.H>
+#include <AMReX_SIMD.H>
+#include <ablastr/constant.H>
```

Comment:
I need to clarify that ABLASTR goes before AMReX [includes](https://impactx.readthedocs.io/en/latest/developers/repo_organization.html#c-includes), because it depends on the latter.

```suggestion
#include <ablastr/constant.H>

#include <AMReX_Extension.H>
#include <AMReX_Math.H>
#include <AMReX_REAL.H>
#include <AMReX_SIMD.H>

```